invenio_rdm_records/services.py

```diff
@@ -8,15 +8,33 @@
 
 """Bibliographic Record Service."""
 
-from invenio_records_resources.services import RecordService, \
-    RecordServiceConfig
+from invenio_drafts_resources.services import RecordDraftService, \
+    RecordDraftServiceConfig
+from invenio_records_resources.services import MarshmallowDataValidator
 
-# TODO: IMPLEMENT ME if more specialized configuration needed
-# class BibliographicRecordServiceConfig(RecordServiceConfig):
-#     """Bibliographic Record Service configuration."""
+from .marshmallow import MetadataSchemaV1
+from .models import BibliographicRecord, BibliographicRecordDraft
+from .permissions import RDMRecordPermissionPolicy
 
 
-class BibliographicRecordService(RecordService):
-    """Bibliographic Record Service."""
+class BibliogrpahicRecordDraftServiceConfig(RecordDraftServiceConfig):
+    """Bibliografic record draft service config."""
 
-    default_config = RecordServiceConfig
+    # Record
+    record_cls = BibliographicRecord
+    permission_policy_cls = RDMRecordPermissionPolicy
```

Comment:
I was actually doing that locally now, in my attempt to integrate `records-resources` :) Nice job :)